lib/tessel/deployment/rust.js

```diff
@@ -53,7 +53,7 @@ exportables.preBundle = function(opts) {
     return Promise.resolve();
   } else {
     // Check that rust, the sdk, and the binary name are valid.
-    return rust.checkRust()
+    return rust.checkRust(true)
```

Comment:
Oof—we eliminated all of the "boolean trap" parameters! Because `rust.checkRust(true)` means nothing useful :\ Does `rust.checkRust(false)` _not_ "check rust"?